concepticondata/conceptlists/Vergallito-2020-1121.tsv

```diff
@@ -0,0 +1,1122 @@
+Vergallito-2020-1121-905	905	scooter	scooter			7	n	0.8333333333	655.56	1	583.48	0.2254230522	0.2001129728	4.25	0.04	3.86	1.74	4.75	v	32.17213115	4.75	0.04	2.928	7.652045478
+Vergallito-2020-1121-906	906	scoraggiato	discouraged			11	a	0.9666666667	725.862069	1	605.3214286	0.5962545098	0.3695192008	3.32	0.12	0.36	0.14	3.54	v	45.72192513	3.54	0.12	1.496	4.870071868
+Vergallito-2020-1121-907	907	scorbuto	scurvy			8	n	0.7666666667	811	0.9642857143	596.44	0.9393601908	0.4506830614	1.36	0.34	1.04	0.34	3.13	v	44.92753623	3.13	0.34	1.242	3.599902776
+Vergallito-2020-1121-908	908	scorie	waste	79	WASTE	6	n	0.6333333333	789.8421053	1	577.6	0.9167642957	0.09416351559	0.85	0.37	1.65	2.07	3.51	v	37.15976331	3.51	0.37	1.69	4.492983419
+Vergallito-2020-1121-909	909	scorpione	scorpion	1538	SCORPION	9	n	1	605.5	1	524.2962963	-0.1831707915	-0.2526344656	0.79	0.37	3.09	0.11	4.68	v	50.55309735	4.68	0.11	1.808	5.67658348
+Vergallito-2020-1121-910	910	scrittore	writer			9	n	1	596.2	1	608.6428571	-0.2601771667	0.5628473518	2.37	0.07	1.42	0.25	4.3	v	50.29726516	4.3	0.07	1.682	5.117685024
+Vergallito-2020-1121-911	911	scrivania	desk			9	n	1	619.8666667	1	557.1428571	-0.006955739006	0.2221462838	0.77	0.04	4.28	0.32	4.77	v	46.46365422	4.77	0.04	2.036	6.462832196
+Vergallito-2020-1121-912	912	sculacciata	spanking			11	n	0.9333333333	839.3571429	1	712.0666667	1.321424378	0.9757347626	3.54	0.02	4.35	0.09	4.35	h	35.06072874	4.35	0.02	2.47	7.098246262
+Vergallito-2020-1121-913	913	scusa	excuse			5	n	0.9666666667	568.5517241	1	526.8148148	-0.4482517236	-0.1999698242	4.12	0.07	0.6	0.09	2.56	a	54.43548387	4.12	0.07	1.488	4.888864899
+Vergallito-2020-1121-914	914	sdegnoso	disdainful			8	a	0.7666666667	834.7826087	1	641.2692308	1.242183753	0.4914868844	3.16	0.52	0.62	0.55	3.27	v	33.86699507	3.27	0.52	1.624	4.651429888
+Vergallito-2020-1121-915	915	sedia	chair	1476	CHAIR	5	n	0.9333333333	574.9285714	1	485.6	-0.4464222358	-0.4162689001	1.37	0.05	4.42	0.3	4.7	v	42.89667897	4.7	0.05	2.168	6.602711564
+Vergallito-2020-1121-916	916	sedile	seat			6	n	0.8333333333	610.44	1	501.5416667	-0.03034008515	-0.2250201702	0.39	0.04	4.14	0.56	4.68	v	47.29867482	4.68	0.04	1.962	6.28564237
+Vergallito-2020-1121-917	917	seno	breast	1402	BREAST	4	n	0.9666666667	576.5517241	1	499.1538462	-0.3787061508	-0.3415413473	0.37	0.81	4.49	1.14	4.81	v	38.20998279	4.81	0.37	2.324	6.737121047
+Vergallito-2020-1121-918	918	sentimento	sentiment			10	n	0.9666666667	685.9310345	1	562.6428571	0.3213642538	0.1683126437	2.42	0.89	1.42	0.88	2.46	v	19.57868649	2.46	0.88	1.614	3.935848066
+Vergallito-2020-1121-919	919	sepoltura	burial			9	n	0.9666666667	689.7241379	1	595.92	0.4125723911	0.2517481064	1.61	0	1.18	0.84	4.44	v	55.01858736	4.44	0	1.614	4.940010121
+Vergallito-2020-1121-920	920	serbatoio	tank	3102	TANK	9	n	1	699.3	1	588.8928571	0.4862218117	0.370682179	0.96	0.02	2.89	1.65	4.51	v	44.76570289	4.51	0.02	2.006	5.686536732
```

Comment:
Wrong, it is "tank" in the sense of "liquid container", not the military vehicle as in Concepticon's TANK